Fluid/Ast/IncludeStatement.cs

```diff
@@ -50,10 +50,14 @@ public IncludeStatement(Expression path, IList<AssignStatement> assignStatements
 
             if (AssignStatements != null)
             {
+                context.EnterChildScope();
```

Comment:
You need to grab the result and pass it to `assignStatement.WriteToAsync`

Reply:
I didn't get your point, what I understand is to make the assignments executed within the scope, similar to https://github.com/sebastienros/fluid/blob/dev/Fluid/Ast/IncludeStatement.cs#L51-L65

Correct me if I'm wrong

Reply:
I was wrong ;)

Reply:
I just didn't remember what I had done :blush:

Reply:
😄